### PR TITLE
Sumayyah-auth-ii

### PR DESCRIPTION
**Pull request by sumi419**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          const token = generateToken(user);
+          res.status(201).json(token);
+        });
+    })
+    .catch((error) => {
+      res.status(500).json({ error });
+    });
+});
+
+const secret = 'I love icecream!';
```
Comment (bdurb) [inline index.js]: I know for a fact that this is not a secret...  👍

File: migrations/20180815142244_users.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex, Promise) {
```
Comment (bdurb) [inline migrations/20180815142244_users.js]: 👍

**Comment by bdurb** [comment]
@sumi419 thank you for the pr

File: index.js (hunk 1 above)
Comment (sumi419) [reply index.js]: 😂

Comment (sumi419) [reply index.js]: 🍨

**Pull request closed without merging**